wordpress-mobile/WordPress-Android
# Refactor login prologue screen

Fixes #19841 

<img width=320 src=https://github.com/wordpress-mobile/WordPress-Android/assets/990349/08a96423-bace-4d20-8916-f2b53a8b51d8 />


-----

## To Test:

- Install Jetpack app
- `Logout` if you're already logged in or Fresh install
- `Verify` Prologue screen is displayed at the start as in image above
- `Tap` on **Continue with WordPress.com** button
- `Verify`, able to login as before
- `Logout` if you're already logged in or Fresh install
- `Tap` on **Enter your existing site address**
- `Verify`, able to login as before
- `Test` WordPress app besides Jetpack app to ensure login is working

-----

## Regression Notes

1. Potential unintended areas of impact

    - No intended side effects

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Tested various login flows

3. What automated tests I added (or what prevented me from doing so)

    - TBD

-----

## PR Submission Checklist:
